jobs/mongodb_backup_job.go

```diff
@@ -147,19 +186,95 @@ func createDBURL(dbConfig DBConnConfig) *url.URL {
 	}
 }
 
-func (j *MongoDBBackupJob) startBackup(ctx context.Context) error {
+func (j *MongoDBBackupJob) startBackup(ctx context.Context) (pbmBackup, error) {
 	j.l.Info("Starting backup.")
 
 	nCtx, cancel := context.WithTimeout(ctx, cmdTimeout)
 	defer cancel()
 
-	output, err := exec.CommandContext(nCtx, pbmBin, "backup", "--mongodb-uri="+j.dbURL.String()).CombinedOutput() // #nosec G204
+	var stderr bytes.Buffer
+	var stdout bytes.Buffer
+	cmd := exec.CommandContext(nCtx, pbmBin, "backup", "--out=json", "--mongodb-uri="+j.dbURL.String()) // #nosec G204
+	cmd.Stderr = &stderr
+	cmd.Stdout = &stdout
+	if err := cmd.Run(); err != nil {
+		return pbmBackup{}, errors.Wrapf(err, "pbm backup error: %s", stderr.String())
+	}
```

Comment:
It seems that you can use `Output` here instead of `CombinedOutput`. We need stderr just to fill error message, and it looks like `Output` does it automaticaly

Reply:
there is `Run()` where you can see `CombinedOutput()`?

But probably you are right, i can use `Output()` but in case of error, type assert it to ExitError and extract error from it

Reply:
`CombinedOutupt` was here before you added `Run`